src/main/java/org/betonquest/betonquest/quest/event/DoNothingPlayerlessEvent.java

```diff
@@ -0,0 +1,20 @@
+package org.betonquest.betonquest.quest.event;
+
+import org.betonquest.betonquest.api.quest.event.StaticEvent;
+
+/**
+ * Playerless event that does nothing. It can be used as a placeholder when playerless event execution isn't an error
+ * in itself or in the case that explicitly nothing should happen.
+ */
+public class DoNothingPlayerlessEvent implements StaticEvent {
+    /**
+     * Create a playerless event placeholder that doesn't do anything.
+     */
+    public DoNothingPlayerlessEvent() {
+    }
+
+    @Override
+    public void execute() {
+        // null object pattern
```

Comment:
I think you should add a (debug) log event instead of really nothing here. This can help to identify the reason it does not work.
@Wolf2323 what is your opinion?

Reply:
I agree